HSDO-637 #Disables quick steps and webauth blocks

pookmish opened the pull request.

pookmish committed:
HSDO-637 #Disables quick steps and webauth blocks

sherakama commented on Stanford/JumpstartLab/Install/Block/BlockSettings.php:
```diff
@@ -139,6 +139,19 @@ public function execute(&$args = array()) {
       $update->execute();
     }
 
+    $disable = array(
```
This functionality is in https://github.com/SU-SWS/stanford_install_tasks/blob/7.x-1.x/Stanford/Jumpstart/Install/Block/BlockSettings.php 

It looks like it might be called too early as the modules may not yet be enabled. Can you revise and refactor the existing task so we don't have the duplication? Thanks.

It looks like the profile is calling this step

sherakama commented on Stanford/JumpstartLab/Install/PathologicSettings.php:
```diff
@@ -0,0 +1,73 @@
+  }
+
+  /**
+   * [requirements description]
```
Please finish.

sherakama commented:
A couple of comments for you and a few code climate fixes.

pookmish committed:
HSDO-637 #removed block disable

pookmish commented:
Nice. didnt know about that one. added block settings and moved to bottom of install profile.
https://github.com/SU-SWS/stanford_sites_jumpstart_lab/commit/7737be5b29508f8fa81d84742e829d24b61935cc
